ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java

```diff
@@ -32,9 +32,8 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore//ignore due to moving forms functionality
     @Test
-    public void EditFormTest() throws Exception {
+    public void EditFormTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
i will have to close this and make a better pr for this ticket

Reply:
Noticed :)